assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -83,21 +83,25 @@ docker.connection.tcp.read_timeout_ms=600000
 
 # This archive contains the server to run the workspace agent and any custom extensions.
 # Che injects this archive into machines when they are booted or started.
-machine.server.ext.archive=${che.home}/lib/ws-agent.zip
+machine.server.ext.archive=${che.home}/lib/ws-agent.tar.gz
 
 # The machine's log files are stored here
 machine.logs.location=${che.logs.dir}/machine/logs
 
 # Size of the machine by default.  What is used if RAM parameter not provided by user or API.
 machine.default_mem_size_mb=1024
 
-# When the workspace master launches a new workspace, Che performs checks of the internal Web 
+# When the workspace master launches a new workspace, Che performs checks of the internal Web
 # services. When Che gets a valid response, we know that the workspace agent is ready for use.
-machine.ws_agent.max_start_time_ms=60000
+machine.ws_agent.max_start_time_ms=120000
```

Comment:
Make sure that user will have nice friendly error message if this timeout occur.